sounds/views.py

```diff
@@ -73,11 +73,9 @@ def get_random_sound():
     cache_key = "random_sound"
     random_sound = cache.get(cache_key)
     if not random_sound:
-        random_sound = Sound.objects.random()
+        rnd = RandomSound.objects.order_by('created').last()
```

Comment:
Part of my comments in #839 was suggesting that we create sounds of the day in advance, so that if we "forget" to run this script, we still have some sounds for the next few days.